pkg/combustion/network.go

```diff
@@ -19,10 +19,10 @@ const (
 	// Used for both input component source and
 	// output configurations subdirectory under combustion.
 	networkConfigDir        = "network"
-	networkConfigScriptName = "configure-network.sh"
+	networkConfigScriptName = "03-configure-network.sh"
```

Comment:
This will now always be the first script we execute. I intentionally didn't go for 01 or 02 in case we'd want to plug in something more important before this but I'm open to changing it as well.

Reply:
Good to leave 01/02 free... 03 is good as we really need to think about bringing up the network really early in the process given the other network-related items that are configured.

Reply:
Agreed that we want to leave some way to inject things at the very start. I still think we need a more comprehensive numbering review once we get close to MVP, though I don't see this one moving from the start.